http/stats.md

```diff
@@ -0,0 +1,54 @@
+|--------------------|--------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| started            | sum                      | Number of all client requests started                                                                                                                                                                                                                                                             |
+| bytes_sent         | sum                      | Total bytes sent in request body (not including headers). This is uncompressed bytes.                                                                                                                                                                                                             |
+| bytes_recv         | sum                      | Total bytes received in response bodies (not including headers but including error responses with bodies). Should be measured from actual bytes received and read, not the value of the Content-Length header. This is uncompressed bytes. Responses with no body should record 0 for this value. |
+| headers_sent       | none                     | Total number of header lines sent in outgoing requests, not including trailing headers                                                                                                                                                                                                            |
+| headers_recv       | none                     | Total number of header lines received in responses, not including trailing headers                                                                                                                                                                                                                |
+| latency            | distribution             | Time between first byte of request headers sent to last byte of response received, or terminal error                                                                                                                                                                                              |
+| ttfb               | distribution             | Time between last byte of request sent and first byte of response body received                                                                                                                                                                                                                   |
```

Comment:
This is very hard to measure correctly, I would remove it or put an optional for this.

Reply:
Go has a good way to measure this (httptrace) and often it's a useful metric. I will comment at the top of the file that not all libraries provide all stats.

Reply:
Please comment.

Reply:
Actually after looking into this further, I think I agree with you that it's not that useful and has definitional problems. I will remove it.